requirements.txt

```diff
@@ -1,5 +1,7 @@
+b64==0.4
```

Comment:
Why are these news requirement being added? I don't see where are we using them

Reply:
Starting from an empty environment, started running the program, installing one by one the required dependencies.
The required ones where
- requests
- PIL (for Pillow)
- b64 for beautifulSoup
- cairosvg 

After that did `pip3 freeze > requirements.txt` and this was the output. I am guessing they are peer dependencies from the other libraries